src/App.js

```diff
@@ -16,6 +17,9 @@ export default function App() {
             <StyledLi>
               <StyledLink to="/">Home</StyledLink>
             </StyledLi>
+            <StyledLi>
+              <SearchComponent />
+            </StyledLi>
             <StyledLi>
               <StyledLink to="/user/:userid">User</StyledLink>
```

Comment:
this will not work, you will go to /user/:userid, you have to add a real id in the link